lib/docs/index.html

```diff
@@ -758,7 +758,7 @@ <h2><span>Zig Version</span></h2>
         <div class="wrap">
           <section class="docs" style="padding-top: 1.5rem; padding-bottom:0;">
           <div style="position: relative">
-            <span id="searchPlaceholder"><kbd>s</kbd> to search, <kbd>?</kbd> for more options</span>
+            <span id="searchPlaceholder"><kbd>/</kbd> or <kbd>s</kbd> to search, <kbd>?</kbd> for more options</span>
```

Comment:
You also have to update a line
```
  879           <dl><dt><kbd>s</kbd></dt><dd>Focus the search field</dd></dl>
```
way down and change "s" to "/" (again, I recommend just removing support for <kbd>s</kbd>).

Reply:
Done